projects/ziti-console-lib/src/lib/assets/scripts/chart.min.js

```diff
@@ -1,7 +1,23 @@
+/*
```

Comment:
should minified files even be in source control? also this isn't our file to add the copyright to, right? are there others like this?

Reply:
good catch, thanks.

I removed from thirdparty libs.

"should minified files even be in source control" probably not.  this was in legacy code and we haven't done much to vette since we are re-writing